### PR TITLE
Simple Refinaments and fixes for Pocillo Theme

### PR DESCRIPTION
**Pull request by UltraBurstXD**
Hi,
I tweaked the theme a little and fixed focus problem when renaming Nemo folder or files.

### CODE REVIEW

File: src/gtk-3.0/3.22/sass/widgets/_scrollbar.scss [hunk 1]
```diff
@@ -30,9 +30,9 @@ scrollbar {
     background-clip: padding-box;
     background-color: $scrollbar_color;
 
-    &:hover { background-color: $fg_color; }
+    &:hover { background-color: $Scrollbar_fg_color; }
```
Comment (fossfreedom) [inline src/gtk-3.0/3.22/sass/widgets/_scrollbar.scss:33]: lowercase name

File: src/gtk-3.0/3.22/sass/widgets/_scrollbar.scss [hunk 2]
```diff
@@ -30,9 +30,9 @@ scrollbar {
...
 
-    &:active { background-color: $fg_color; }
+    &:active { background-color: $Scrollbar_fg_color2; }
```
Comment (fossfreedom) [inline src/gtk-3.0/3.22/sass/widgets/_scrollbar.scss:35]: lowercase name

File: src/gtk-3.0/3.22/sass/_colors.scss [hunk 3]
```diff
@@ -21,13 +21,15 @@ $light_fg_color:  rgba($white, 0.85);
 $button_fg_color: if($variant == 'light', rgba($brown_800, 0.75), rgba($white, 0.8)); // extra fg color for raised buttons
 
 $fg_color:                    if($variant == 'light', rgba($brown_800, 0.85), $white);
+$Scrollbar_fg_color:          if($variant == 'light', rgba($grey_500, 0.95), rgba($grey_200, 0.75));
+$Scrollbar_fg_color2:         if($variant == 'light', rgba($grey_600, 0.75), rgba($grey_300, 0.85));
```
Comment (fossfreedom) [inline src/gtk-3.0/3.22/sass/_colors.scss:25]: lowercase names

Also better name would be scrollbar_active_fg_color for Scrollbar_fg_color2  to be consistent with other color variable names

Comment (UltraBurstXD) [reply src/gtk-3.0/3.22/sass/_colors.scss]: Alright, fixed the little (unnoticed :sweat_smile: ) typo.

**Comment by fossfreedom** [comment]
Many thanks - all looks good from a code review.  I will do a cursory check with a build of your suggestions to double check dark/normal/light and slim variants asap.

**Comment by UltraBurstXD** [comment]
My pleasure.

**Pull request merged**